### PR TITLE
Memory1

### PR DESCRIPTION
**Pull request by tinytort**

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    return storesList;
+}
+
+function remove(_id) {
+    const removingObj = storesList.find(x => x._id === _id);
+    if(!removingObj) {
+        return {removed: false};
+    } else {
+        let index = storesList.indexOf(removingObj);
+        storesList.splice(index, 1);
```
Comment (nsmeds) [inline src/store.js]: Your `if` block returns `{removed: false}` - make sure your `else` block returns something as well.

File: test/store.test.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    });
+});
+
+describe('memory store', () => {
+    describe('remove', () => {
+        it('removes a specified object from the array', () => {
+            const testRemoveObj = {foo: 'bar'};
+            const testObjArray = store.save(testRemoveObj);
+            const removeFromArray = store.remove(testRemoveObj);
+            assert.ok({remove: true});
```
Comment (nsmeds) [inline test/store.test.js]: There are a couple of problems with this test. First, your assertion only tests that the string `remove` is truthy. You could substitute any string for `remove` and still get a passing test. I would make sure your test references the data coming back from your app. This is a good place to use the debugger and take a close look at your `const`s and what values they're producing.

File: test/store.test.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+});
+
+describe('memory store', () => {
+    describe('get', () => {
+        it('gives the id for an obj', () => {
+            const testObj = { foo: 'bar' };
+            const savedObj = store.save(testObj);
+            const foundObj = store.get(savedObj._id);
+            assert.deepEqual(savedObj, foundObj);
+        });
```
Comment (nsmeds) [inline test/store.test.js]: Make sure you're testing for (and creating) all the functionality specified in the [LAB.md](https://github.com/tinytort/in-memory-object-store/blob/master/LAB.md) - here you need to also return `null` if that id does not exist.

File: test/store.test.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+});
+
+describe('memory store', () => {
+    describe('getAll', () => {
+        it('returns an array of the objects', () => {
+            const testfirstObj = { foo: 'bar' };
+            const savedfirstObj = store.save(testfirstObj);
+            let newArray = store.getAll();
+            assert.deepEqual(store.storesList, newArray);
+        });
```
Comment (nsmeds) [inline test/store.test.js]: As above, make sure you also test that your getAll returns an empty array if no objects exist.
